src/translations.js

```diff
@@ -19,6 +19,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       np: `${encodeHTML(name)}'${apostrophe} गिटहब तथ्याङ्क`,
       el: `Στατιστικά GitHub του ${encodeHTML(name)}`,
       ru: `Статистика GitHub пользователя ${encodeHTML(name)}`,
+      "uk-ua": `Статистика GitHub користувача ${encodeHTML(name)}`,
```

Comment:
I believe Ukrain's language code is `uk` 🤔

Reply:
<img width="711" alt="Screenshot 2021-01-07 at 16 54 56" src="https://user-images.githubusercontent.com/477114/103900651-15f70d80-5109-11eb-9f5f-fdba04bfdb47.png">

Reply:
Ohh okay got it! thanks.